specification/maps/data-plane/Microsoft.Maps/Alias/preview/2.0/alias.json

```diff
@@ -68,7 +68,7 @@
       }
     },
     "401": {
-      "description": "Access denied due to invalid subscription key or invalid Azure Active Directory (Azure AD) bearer token.  Make sure to provide a valid key for an active Azure subscription and Maps resource.  Otherwise, verify the [WWW-Authenticate](https://tools.ietf.org/html/rfc6750#section-3.1) header for error code and description of the provided Azure AD bearer token.",
+      "description": "Access denied due to invalid subscription key or invalid  Microsoft Entra ID bearer token.  Make sure to provide a valid key for an active Azure subscription and Maps resource.  Otherwise, verify the [WWW-Authenticate](https://tools.ietf.org/html/rfc6750#section-3.1) header for error code and description of the provided Microsoft Entra ID bearer token.",
```

Comment:
There's an extra space in front of the first "Microsoft Entra ID". Same for all other files.

Reply:
fixed